data/data-pipeline/data_pipeline/etl/sources/geo_utils.py

```diff
@@ -18,38 +19,66 @@ def get_tract_geojson(
     GEOJSON_PATH = _tract_data_path
     if GEOJSON_PATH is None:
         GEOJSON_PATH = CensusETL.NATIONAL_TRACT_JSON_PATH
-        if not GEOJSON_PATH.exists():
-            logger.debug("Census data has not been computed, running")
-            census_etl = CensusETL()
-            census_etl.extract()
-            census_etl.transform()
-            census_etl.load()
-        else:
-            logger.debug("Loading existing tract geojson")
-    tract_data = gpd.read_file(GEOJSON_PATH, include_fields=["GEOID10"])
-    tract_data.rename(columns={"GEOID10": "GEOID10_TRACT"}, inplace=True)
+    if not GEOJSON_PATH.exists():
+        logger.debug("Census data has not been computed, running")
+        census_etl = CensusETL()
+        census_etl.extract()
+        census_etl.transform()
+        census_etl.load()
+    tract_data = gpd.read_file(
+        GEOJSON_PATH,
+        include_fields=["GEOID10"],
+    )
+    tract_data = tract_data.rename(
+        columns={"GEOID10": "GEOID10_TRACT"}, errors="raise"
+    )
     return tract_data
 
 
+@lru_cache()
```

Comment:
I am curious about the usage of `lru_cache` here. Is this function called multiple times?

Reply:
I am guessing he was following the convention `get_tract_geojson`, and i was like "well, it won't hurt, since it is expensive. I believe at this time it isn't, so I'm good to drop the cache untel we need ot/

Reply:
It is actually useful on get_tract_geojson, so I vote keep it. And I'm about to merge so it's stayin'